reactjs-films-homework/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react';
+import styles from './ErrorBoundary.module.scss';
+
+interface Props {
+  children: ReactNode;
+  title: string;
+}
+
+interface State {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  public state: State = {
+    hasError: false,
+  };
+
+  public static getDerivedStateFromError(_: Error): State {
+    return { hasError: true };
+  }
+  public componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+    console.error('Uncaught error:', error, errorInfo);
+  }
+
+  public render() {
+    if (this.state.hasError) {
+      return (
+        <div className={styles.boundaryWrapper}>
+          <h2 color="wite">
```

Comment:
Please use classes for styling. A typo here BTW, should be 'white' :)

Reply:
done !)